leasing/management/commands/attach_areas.py

```diff
@@ -22,28 +24,37 @@ class Command(BaseCommand):
     def handle(self, *args, **options):  # noqa: C901 TODO
         from auditlog.registry import auditlog
 
+        logging.basicConfig(level=logging.INFO, format="%(message)s", stream=sys.stdout)
+        log = logging.getLogger(__name__)
+
+        verbosity = options.get("verbosity")
+        if verbosity == 0:
+            log.setLevel(logging.WARNING)
+        elif verbosity == 2:
+            log.setLevel(logging.DEBUG)
```

Comment:
Use verbosity >= 2 here, since the help text mentions also 3.

Reply:
Fixed